src/modules/users/components/TokenActivation/TokenActivationContent/ChangeTokenStateForm.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useCallback, useState } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { BigNumber, bigNumberify } from 'ethers/utils';
+import { FormikProps } from 'formik';
+import moveDecimal from 'move-decimal-point';
+import * as yup from 'yup';
+
+import Button from '~core/Button';
+import { ActionForm, Input } from '~core/Fields';
+import Numeral from '~core/Numeral';
+
+import { UserToken } from '~data/generated';
+import { ActionTypes } from '~redux/index';
+import { pipe, mapPayload } from '~utils/actions';
+
+import styles from './TokenActivationContent.css';
+
+const MSG = defineMessages({
+  changeState: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.changeState',
```

Comment:
This should be changed from `.TokensTab.` from to`ChangeTokenStateForm` right?

Reply:
Same with the other message ids

Reply:
good catch! :)